packages/wallet/backend/src/gatehub/types.ts

```diff
@@ -47,6 +47,7 @@ export interface IGetUserStateResponse {
     address_street1: string
     address_street2: string
   }
+  meta: Record<string, string>
```

Comment:
We should ask to receive this field on fetching the user state if we don't already (cannot find it in the docs though)

Reply:
There is no `GET` options for a user?